extensions/positron-assistant/package.json

```diff
@@ -329,7 +329,7 @@
       {
         "name": "inspectVariables",
         "displayName": "Inspect Variables",
-        "modelDescription": "List the children of an array of variables in a session. For example, the columns in a dataframe, items in a column or array, or elements of a list. If `accessKeys` is empty, lists all root-level variables in the session.\n\nIf the user references a variable by name, first determine the `access_key` from the user context or a previous inspect variables result.\n\nDo not call when:\n  - the variables do not appear in the user context\n  - there is no session (the session identifier is empty or not defined)",
+        "modelDescription": "List the children of an array of variables in a session. For example, the columns in a dataframe, items in a column or array, or elements of a list. If `accessKeys` is empty, lists all root-level variables in the session.\n\nIf the user references a variable by name, first determine the `access_key` from the user context or a previous inspect variables result.",
```

Comment:
- removed `Do not call when: the variables do not appear in the user context` since this didn't seem to be respected by the models anyways
- removed `Do not call when: there is no session (the session identifier is empty or not defined)` since it is covered in https://github.com/posit-dev/positron/issues/8078